tools/iptb-plugins/filecoin/docker/iptb_metrics.go

```diff
@@ -14,30 +14,34 @@ func (l *Dockerfilecoin) Events() (io.ReadCloser, error) {
 
 // StderrReader provides an io.ReadCloser to the running daemons stderr
 func (l *Dockerfilecoin) StderrReader() (io.ReadCloser, error) {
-	cli, _ := l.GetClient()
-	ctx, _ := context.WithCancel(context.Background()) // nolint: vet
+	cli, err := l.GetClient()
+	if err != nil {
+		return nil, err
+	}
 
 	options := types.ContainerLogsOptions{
 		ShowStdout: false,
 		ShowStderr: true,
 		Follow:     true,
 	}
 
-	return cli.ContainerLogs(ctx, l.ID, options)
+	return cli.ContainerLogs(context.Background(), l.ID, options)
```

Comment:
We are no longer making the context cancel-able here. Is this a conscious change? Do we know why it was cancel-able in the first place?

Reply:
Since we were ignoring the `cancel()` function returned by `context.WithCancel()` and since it didn't have a parent context I don't think we were ever canceling it. I believe this change is alright.

Reply:
I think it just slipped through code review when this was originally merged. It started out as a cancel-able context to implement the Closer interface for the io.ReadCloser, turns out that ContainerLogs already returns a io.ReadCloser so it was no longer needed to shutdown the stream.